packages/pirania-app/files/etc/pirania/governance.json

```diff
@@ -0,0 +1,18 @@
+{
+  "provider": {
```

Comment:
can you describe the structure of this file? it is not clear what provider, community and member are, and how this affects the usage.
would be good to have a general documentaiton

Reply:
Will do.